src/estimator.cpp

```diff
@@ -139,7 +138,7 @@ void Estimator::run()
   state_.timestamp_us = now_us;
 
   // Crank up the gains for the first few seconds for quick convergence
-  if (now_us < (uint64_t)RF_.params_.get_param_int(PARAM_INIT_TIME)*1000)
+  if (now_us < static_cast<uint64_t>(RF_.params_.get_param_int(PARAM_INIT_TIME)*1000))
```

Comment:
Should the `*1000` be outside of the cast to `uint64_t`? Just wondering if there's possibility for overflow here if we do the multiplication on a `int32_t`